setup.py

```diff
@@ -44,6 +44,7 @@
         'funcy',
         'semantic-version',
         'jsonschema>=3.2.0',
+        'google-cloud-storage',
```

Comment:
will require reinstalling venv. Please send an email to OCS-QE about it once this PR is merged